service/history/api/describeworkflow/api.go

```diff
@@ -143,6 +143,15 @@ func Invoke(
 		},
 	}
 
+	// copy pause info to the response if it exists
+	if executionInfo.PauseInfo != nil {
+		result.WorkflowExecutionInfo.PauseInfo = &workflowpb.WorkflowExecutionPauseInfo{
```

Comment:
WorkflowExecutionInfo is used by visibility as well. Shall we add PauseInfo to WorkflowExtendedInfo instead?